src/main/java/blackjack/domain/participant/Hand.java

```diff
@@ -0,0 +1,56 @@
+package blackjack.domain.participant;
+
+import blackjack.domain.card.Card;
+import blackjack.domain.card.CardLetter;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Hand {
+    private static final int BUST_LIMIT = 22;
+    private static final int NO_ACE = 0;
+
+    private final List<Card> cards;
+
+    public Hand() {
+        this.cards = new ArrayList<>();
+    }
+
+    public void add(final Card card) {
+        cards.add(card);
+    }
+
+    public boolean isBust() {
+        return calculateScore() >= BUST_LIMIT;
+    }
+
+    public int calculateScore() {
+        final int maximumSum = cards.stream()
+                .mapToInt(card -> card.getCardLetter().getValue())
+                .sum();
+
+        if (maximumSum >= BUST_LIMIT && countAce() > NO_ACE) {
+            return adjustScoreWithAce(maximumSum);
+        }
+        return maximumSum;
+    }
+
+    private int countAce() {
+        return (int) cards.stream().filter(Card::isAce).count();
+    }
```

Comment:
```suggestion
    private int countAce() {
        return (int) cards.stream()
            .filter(Card::isAce)
            .count();
    }
```

개행으로 가독성을 높일 수 있겠네요! :)

Reply:
네 적용했습니다~